res/values/strings.xml

```diff
@@ -599,17 +599,17 @@
     <string name="global_menu_edit_copy_desktop">Copy</string>
     <string name="global_menu_edit_paste_desktop">Paste</string>
     <string name="global_menu_view_desktop">View</string>
-    <string name="global_menu_view_floatontop_desktop">Float on Top</string>
+    <string name="global_menu_view_floatontop_desktop">Float-on-top</string>
```

Comment:
Float atop what?

Reply:
on top of all other windows, this is is a usual term in the view menu of many desktop-apps.

Reply:
Isnt that "Always above (other windows)" or similar?